### PR TITLE
Fix/tec 3009 start end time same

### PR DESCRIPTION
**Pull request by paulmskim**
[TEC-3009]

Screencast: https://drive.google.com/open?id=1amuxu6xMDPKLyivpzudXzXHTQP6Eg2oo

[TEC-3009]: https://moderntribe.atlassian.net/browse/TEC-3009

### CODE REVIEW

File: src/views/blocks/event-datetime.php [hunk 1]
```diff
@@ -80,34 +81,36 @@
...
-		<?php endif; ?>
-
-		<?php if ( ! $is_same_day ) : ?>
-			<span class="tribe-events-schedule__date tribe-events-schedule__date--end">
-				<?php echo $formatted_end_date; ?>
-			</span>
+		<?php if ( ! $is_same_start_end ) : ?>
+			<?php if ( ! $is_all_day || ! $is_same_day ) : ?>
+				<span class="tribe-events-schedule__separator tribe-events-schedule__separator--time">
+					<?php echo $separator_time; ?>
```
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$separator_time'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$separator_time'.

<sub>hash: 10fbd97efdabbac037705ce58622e6d7</sub>

File: src/views/blocks/event-datetime.php [hunk 2]
```diff
@@ -80,34 +81,36 @@
...
+				</span>
+			<?php endif; ?>
 
-			<?php if ( ! $is_all_day ) : ?>
-				<span class="tribe-events-schedule__separator tribe-events-schedule__separator--date">
-					<?php echo $separator_date; ?>
+			<?php if ( ! $is_same_day ) : ?>
+				<span class="tribe-events-schedule__date tribe-events-schedule__date--end">
+					<?php echo $formatted_end_date; ?>
```
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$formatted_end_date'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$formatted_end_date'.

<sub>hash: f0851e888dd702e699b248649e94a3c9</sub>

File: src/views/blocks/event-datetime.php [hunk 3]
```diff
@@ -80,34 +81,36 @@
...
 				</span>
+
+				<?php if ( ! $is_all_day ) : ?>
+					<span class="tribe-events-schedule__separator tribe-events-schedule__separator--date">
+						<?php echo $separator_date; ?>
```
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$separator_date'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$separator_date'.

<sub>hash: fbd14ad3803ba5a5aca194aff45d0a1c</sub>

File: src/views/blocks/event-datetime.php [hunk 4]
```diff
@@ -80,34 +81,36 @@
...
+					</span>
+					<span class="tribe-events-schedule__time tribe-events-schedule__time--end">
+						<?php echo $formatted_end_time; ?>
```
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$formatted_end_time'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$formatted_end_time'.

<sub>hash: c1786c78ed93b8302bb50da6b46db594</sub>

File: src/views/blocks/event-datetime.php [hunk 5]
```diff
@@ -57,8 +57,9 @@
 	$separator_date = tribe_get_option( 'dateTimeSeparator', ' - ' );
 }
 
-$is_all_day       = tribe_event_is_all_day( $event_id );
```
Comment (juanfra) [inline src/views/blocks/event-datetime.php]: Let's set the view version as TBD.

File: src/views/blocks/event-datetime.php (hunk 1 above)
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$separator_time'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$separator_time'.

<sub>hash: 05e6ff5d0ce1390512a68adc80591593</sub>

File: src/views/blocks/event-datetime.php (hunk 2 above)
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$formatted_end_date'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$formatted_end_date'.

<sub>hash: 3ff57ef72d3504a71b66d626104f2381</sub>

File: src/views/blocks/event-datetime.php (hunk 3 above)
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$separator_date'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$separator_date'.

<sub>hash: 3ecf734af43754818ab5e188efadc04d</sub>

File: src/views/blocks/event-datetime.php (hunk 4 above)
Comment (tr1b0t) [inline src/views/blocks/event-datetime.php]: * :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$formatted_end_time'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$formatted_end_time'.

<sub>hash: 566ddd16892514840d24c8abeeababe7</sub>

**Pull request merged**